docs/content/index.md

```diff
@@ -2,60 +2,153 @@
 layout: doc_page
 ---
 
-# About Druid
+Introduction to Druid
+---------------------
 
-Druid is an open-source analytics data store designed for OLAP queries on timeseries data (trillions of events, petabytes of data). Druid provides cost-effective and always-on real-time data ingestion, arbitrary data exploration, and fast data aggregation.
+Druid is an open source data store designed for [OLAP](http://en.wikipedia.org/wiki/Online_analytical_processing) queries on timeseries data.
```

Comment:
I agree about index.md being too long.  I like having the sections about the data (indexing, sharding, loading) because that really answers my first questions about druid.  But the sections after that, starting with the druid cluster, seemed like they belonged on a separate page.